MusicDatabaseGenerator/Generators/AlbumArtGenerator.cs

```diff
@@ -75,5 +77,26 @@ private string ColorToHexString(Color color)
             string b = color.B.ToString("X2");
             return $"#{r}{g}{b}";
         }
+
+        private void UnhideAlbumArtImages(string path)
+        {
+            if (File.GetAttributes(path).HasFlag(FileAttributes.Hidden))
+            {
+                File.SetAttributes(path, FileAttributes.Normal);
+            }
+        }
+
+        /*private void TEMP_ExportImages(string path)
```

Comment:
May be added as a feature later, if users want a copy of just the images in their library. It's what I used to quickly get just the images moved over to my Android device